AsyncDisplayKit/ASRunLoopQueue.mm

```diff
@@ -0,0 +1,98 @@
+//
+//  ASRunLoopQueue.m
+//  AsyncDisplayKit
+//
+//  Created by Rahul Malik on 3/7/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import "ASRunLoopQueue.h"
+#import "ASThread.h"
+
+#import <deque>
+
+@interface ASRunLoopQueue () {
+  CFRunLoopRef _runLoop;
+  CFRunLoopObserverRef _runLoopObserver;
+  std::deque<id> __renderQueue;
```

Comment:
In a later change, you can rename these to single underscore _ for instance variables now that they are not statics